### PR TITLE
Upgrade to cats 1.0.1

### LINKED ISSUE #133: Update to Cats 1.0.0

### PR DESCRIPTION
**Pull request by AdrianRaFo**
With freestyle 0.5.1
Fixes #133

### CODE REVIEW

**Review by dominv — APPROVED** [review]
LGTM

File: core/src/main/scala/schema/provider/metadata/SchemaConversions.scala [hunk 1]
```diff
@@ -61,7 +62,7 @@ trait SchemaConversions {
     E.flatten {
       catchNonFatalAsSchemaError {
         val columnsM: M[List[Table.Column]] =
-          E.traverse(metadata.getColumns.asScala.toList)(toTableColumn[M](_)(E))
+          Traverse[List].traverse(metadata.getColumns.asScala.toList)(toTableColumn(_))(E)
```
Comment (fedefernandez) [inline core/src/main/scala/schema/provider/metadata/SchemaConversions.scala]:
```diff
--          Traverse[List].traverse(metadata.getColumns.asScala.toList)(toTableColumn(_))(E)
++          metadata.getColumns.asScala.toList.traverse(toTableColumn(_)(E))
```

File: core/src/main/scala/schema/provider/metadata/SchemaConversions.scala [hunk 2]
```diff
@@ -18,6 +18,7 @@ package freestyle.cassandra
 package schema.provider.metadata
 
 import cats.MonadError
+import cats.Traverse
```
Comment (fedefernandez) [inline core/src/main/scala/schema/provider/metadata/SchemaConversions.scala]: Not needed

**Review by fedefernandez — CHANGES_REQUESTED** [review]

**Review by fedefernandez — APPROVED** [review]
giphy

**Pull request merged**